### PR TITLE
change `types` import location

### PR DESCRIPTION
**Pull request by LiangJYu**
This PR changes `types` import location from `nisar` to `isce3.core` to reflect changes made in this [PR](https://github-fn.jpl.nasa.gov/isce-3/isce/pull/1397).

### CODE REVIEW

**Review by vbrancat — APPROVED** [review]
LGTM. We should merge this PR once the sister PR on ISCE3 gets merged as well

**Review by seongsujeong — APPROVED** [review]
LGTM. Thanks for the PR to apply the change about the module location.

**Pull request merged**
